src/js/port_handler.js

```diff
@@ -1,5 +1,50 @@
 'use strict';
 
+const http = require("http");
+const bonjour = require('bonjour')();
+
+const MDNS_INTERVAL = 10000;
+const TCP_CHECK_INTERVAL = 5000;
+const TCP_TIMEOUT = 2000;
+
+const mdnsBrowser = {
+    services: [],
+};
+
+function isMobile() {
+    return "Android" === GUI_checkOperatingSystem();
+}
+
+if (isMobile()) {
```

Comment:
```suggestion
if (GUI.isCordova()) {
```

Reply:
This does not work, I had it this way earlier, but when port_handler initializes, the GUI object is not there. This is why I extracted the other function.